### ISSUE TITLE
Option for suggest command to output just the suggested version

### ISSUE THREAD

**Issue by SebSept** [tomzx/php-semver-checker-git#19]
```bash
/php-semver-checker-git.phar suggest  --only-version "./" "./" 
```
So it output only 
```
0.4.0
```

Or maybe the result like : 
```
Testing HEAD against tag: 0.3.1
                                        
Initial semantic version: 0.3.1
Suggested semantic version: 0.4.0
```
should be display only with the -v option passed 

What do you think of that ?

**Comment by tomzx** [comment]
I wouldn't mind, but for what purpose would you need it?

Other than that, it is pretty easy to write a wrapper script that extracts the version out of the final output.

**Comment by SebSept** [comment]
I'd like to replace a placeholder in the code with the php-semver-checker-git result. So Application version is automatically updated.
So a git hook, before commit, can replace that placeholder with the suggested version.
(That's a symfony console app packaged as phar - http://api.symfony.com/master/index.html?q=setVersion )

**Comment by SebSept** [comment]
this can do the trick with awk
```sh
./php-semver-checker-git.phar suggest | awk '/Suggested semantic version: / {print $4}'
```
so maybe this is not a relevant enhancement ...

**Comment by tomzx** [comment]
@SebSept As long as the output of `php-semver-checker-git` does not change, absolutely.

However, I do think there's some value in outputting only the suggested semantic version number. I'll try to figure out an appropriate solution when I have some free time.

**Comment by SebSept** [comment]
Right, it's weak because it rely on the output, that's just a tip.
Thanks :)
